<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AnamikaRao-QA" ignored="10" name="test" tests="6" failures="6" timestamp="30 Aug 2021 09:52:22 GMT" time="0.088" errors="0">
  <testcase name="addData" time="0.014" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.buynowbutton(Testcases.java:66)
at TestCases.Testcases.addData(Testcases.java:95)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- addData -->
  <testcase name="buynowbutton" time="0.002" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.buynowbutton(Testcases.java:66)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- buynowbutton -->
  <testcase name="poup" time="0.003" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.buynowbutton(Testcases.java:66)
at TestCases.Testcases.addData(Testcases.java:95)
at TestCases.Testcases.poup(Testcases.java:107)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- poup -->
  <testcase name="verifyTitle" time="0.006" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.verifyTitle(Testcases.java:59)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <testcase name="verifyTotal" time="0.002" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.buynowbutton(Testcases.java:66)
at TestCases.Testcases.verifyTotal(Testcases.java:75)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyTotal -->
  <testcase name="verifylabel" time="0.001" classname="TestCases.Testcases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.Testcases.buynowbutton(Testcases.java:66)
at TestCases.Testcases.verifylabel(Testcases.java:82)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifylabel -->
  <testcase name="promoCode" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- promoCode -->
  <testcase name="verifyData" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- verifyData -->
  <testcase name="debitCardDetails" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- debitCardDetails -->
  <testcase name="redirecttopayment" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- redirecttopayment -->
  <testcase name="paymentMethod" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- paymentMethod -->
  <testcase name="verifyincorrectpin" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- verifyincorrectpin -->
  <testcase name="veriyCancelbutton" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- veriyCancelbutton -->
  <testcase name="transcationConfirmation" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- transcationConfirmation -->
  <testcase name="verifycorrectpin" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- verifycorrectpin -->
  <testcase name="cardinput" time="0.0" classname="TestCases.Testcases">
    <ignored/>
  </testcase> <!-- cardinput -->
</testsuite> <!-- test -->
